vsts/pipelines/ci.yml

```diff
@@ -1,4 +1,10 @@
 parameters:
+  - name: storageAccountUrl
+    displayName: Storage Account Url
+    type: string
+    default: https://oryx-cdn.microsoft.io
+    values:
+    - https://oryx-cdn.microsoft.io
```

Comment:
Currently means that the Oryx-CI job will _only_ be able to use the prod CDN...

This comes back to the question of whether or not we want to be pre-baking with the dev or prod storage accounts

Reply:
If we do not want this behavior, I will need to add in options to specify a pre-bake storage account and a dynamic installation storage account, as the 2 will need to be able to be different.